api/app/extensions/ldap_extension.py

```diff
@@ -0,0 +1,165 @@
+"""Open LDAP authentication extension."""
```

Comment:
If rewritten from the Python package, you should probably leave a notice here about it

If that package has some license, that is of course important to consider as well

Reply:
Here: https://github.com/ContinuumIO/flask-ldap-login/blob/master/LICENSE

```
* Redistributions of source code must retain the above copyright notice, this
  list of conditions and the following disclaimer.

* Redistributions in binary form must reproduce the above copyright notice,
  this list of conditions and the following disclaimer in the documentation
  and/or other materials provided with the distribution.
```

Reply:
Not sure if this is rewritten code, or "glue" code interfacing with it. I was thinking if it was the first case - that this is rewritten code from that package.

Reply:
I dont know where the difference lie.

Reply:
I think you might have to add the license text into this document as well, but we can discuss at some point